client/src/components/ActionSection.vue

```diff
@@ -45,4 +72,25 @@ export default defineComponent({
 .explain-text {
   padding: $spacing-md;
 }
+
+.icon {
+  max-width: $spacing-md;
+}
+
+// These variables cannot reference css vars
+.tooltip-content {
```

Comment:
Can you use the copper design tokens for any of this?

Reply:
for some reason these css var properties don't work with css vars